src/main/java/Application.java

```diff
@@ -1,6 +1,33 @@
+import Source.WordSource;
+import stdin.StdinCharSource;
+import stdout.AttemptsHangman;
+import stdout.StdoutAnswer;
+
 public class Application {
     public static void main(String[] args) {
-        new GameContext().start();
-    }
+        var wordChoice = new WordSource("/words.txt");
+        wordChoice.readerFile();
```

Comment:
Метод можно назвать более лаконично, например просто `read`, если хочешь оставить постфикс `File`, то тогда: `readFile`.

А readerFile - переводится как "чтец файла".

```suggestion
        wordChoice.read();
```